src/sentry/profiles/task.py

```diff
@@ -87,9 +85,28 @@ def _normalize(
             }
         )
 
+    if profile["platform"] in ["cocoa", "android"]:
+        classification_options.update(
+            {
+                "model": profile["device_model"],
+                "os_name": profile["device_os_name"],
+                "is_emulator": profile["device_is_emulator"],
+            }
+        )
+        profile.update({"device_classification": str(classify_device(**classification_options))})
+
+    if profile["platform"] == "rust":
+        profile.update(
+            {
+                "device_classification": "",
+                "device_locale": "",
+                "device_manufacturer": "",
+                "device_model": "",
+            }
+        )
```

Comment:
Having to do things like this suggests to me that we should look at unifying the underlying table schema to work with the different platforms we want to support soon.